editor.planx.uk/src/@planx/components/Send/model.ts

```diff
@@ -13,14 +15,14 @@ interface EventPayload {
 }
 
 export interface Send extends BaseNodeData {
-  title: string;
+  title?: string;
```

Comment:
nit: can we keep `title` required here please? I have a vague memory of "requiring" all Node titles in the past for analytics or similar?

Reply:
Good spot - fixed in [fbb1d3a](https://github.com/theopensystemslab/planx-new/pull/4911/commits/fbb1d3a7210334e5dffe35f28a34c4834464c20d)